needle/modules/storage/data/files_mdm.py

```diff
@@ -0,0 +1,128 @@
+from core.framework.module import BaseModule
+from core.device.device import Device
+from core.utils.menu import choose_from_list_data_protection, choose_from_list, choose_yes_no
+from core.utils.utils import Utils
+from core.utils.printer import Colors
+
+class Module(BaseModule):
+    meta = {
+        'name': 'MDM dump',
+        'author': 'Oliver Simonnet (@mwr)',
+        'description': 'Locate and dump MDM configuration files',
+        'options': (
+            ('bulk', False, False, 'Dump all configuration files in a selected directory.'),
```

Comment:
have a look at `storage/data/files_plist`. For coherence you should use `DUMP_ALL`